artifactory/.gitignore

```diff
@@ -0,0 +1,2 @@
+results
```

Comment:
`results` is covered by the root `.gitignore`, but if we need additions, i'd prefer seeing these in the root `.gitignore` also.

Reply:
I have removed the `.gitignore`